tests/manage/pv_services/test_expansion_snapshot_clone.py

```diff
@@ -91,18 +100,66 @@ def setup(
             sc_rbd=self.pvc_create_sc,
         )
 
+        self.ct_pod = pod.get_ceph_tools_pod()
+        if pvc_create_sc_type == "thick":
+            self.verify_thick_provision()
+
+    def verify_thick_provision(
```

Comment:
this function will be useful for other test cases as well

Reply:
Modified the implementation